copybookreader-plugins/docs/CopyBookReader-batchsource.md

```diff
@@ -0,0 +1,99 @@
+# CopybookReader plugin
+
+
+Description
+-----------
+This is a source plugin that would allow users to read and process mainframe files defined using COBOL Copybook.
+This is the basic first implementation.
```

Comment:
This is a source plugin that allows users to read and process mainframe files defined using a COBOL Copybook.